uds/__init__.py

```diff
@@ -21,6 +21,7 @@
 __author__ = "Maciej Dąbrowski"
 __maintainer__ = "Maciej Dąbrowski"
 __credits__ = ["Maciej Dąbrowski (https://www.linkedin.com/in/maciej-dabrowski-test-engineer/)",
+               "Przemysław Nieścior (https://www.linkedin.com/in/przemys%C5%82aw-nie%C5%9Bcior-33631021b/)",
```

Comment:
```suggestion
               "Przemysław Nieścior (https://www.linkedin.com/in/przemyslaw-niescior/)",
```
Thank you :heart_hands: